src/app/_components/TherapiesSection.js

```diff
@@ -0,0 +1,46 @@
+import { TopWave } from './TopWave';
+import { BottomWave } from './BottomWave';
+import { TherapyCard } from './TherapyCard';
+import { Heading } from './Typography';
+
+export function TherapiesSection() {
+  return (
+    <section className="relative bg-primary-200 bg-blend-multiply ">
+      <div
+        style={{
+          position: 'absolute',
+          top: 0,
+          left: 0,
+          right: 0,
+          bottom: 0,
+          backgroundImage: "url('/assets/noise.png')",
+          backgroundSize: 'cover',
+          backgroundPosition: 'center',
+          opacity: 0.4,
+        }}
```

Comment:
As I know inline styles are not good practice.
And it is good to avoid them if possible.
https://www.thoughtco.com/avoid-inline-styles-for-css-3466846

Could you just use a class for it?
But if it is not possible - let me know.

Reply:
sure, done